src/test/java/com/hocs/test/pages/data_input/DataInput.java

```diff
@@ -18,121 +18,100 @@
+    public WebElementFacade addMemberOfParliamentSearchField;
 
     @FindBy(css = "label[for='CopyNumberTen-TRUE']")
-    private WebElementFacade sendCopyToNumberTenTickBox;
+    public WebElementFacade sendCopyToNumberTenTickBox;
 
     @FindBy(css = "label[for='AdditionalCorrespondent-FALSE']")
-    private WebElementFacade addCorrespondentNoRadioButton;
+    public WebElementFacade addCorrespondentNoRadioButton;
 
     @FindBy(css = "label[for='AdditionalCorrespondent-TRUE']")
-    private WebElementFacade addCorrespondentYesRadioButton;
+    public WebElementFacade addCorrespondentYesRadioButton;
 
     @FindBy(css = "label[for='CopyNumberTen-TRUE']")
-    private WebElementFacade shouldResponseBeCopiedN10YesRadioButton;
+    public WebElementFacade shouldResponseBeCopiedN10YesRadioButton;
 
     @FindBy(css = "label[for='CopyNumberTen-FALSE']")
-    private WebElementFacade shouldResponseBeCopiedN10NoRadioButton;
+    public WebElementFacade shouldResponseBeCopiedN10NoRadioButton;
 
     @FindBy(xpath = "//a[text()='When was the correspondence sent? is required']")
-    private WebElementFacade correspondenceDateErrorMessage;
+    public WebElementFacade correspondenceDateErrorMessage;
 
     @FindBy(xpath = "//a[text()='How was the correspondence received? is required']")
-    private WebElementFacade howWasCorrespondenceReceivedErrorMessage;
+    public WebElementFacade howWasCorrespondenceReceivedErrorMessage;
 
     @FindBy(xpath = "//a[text()='Which is the primary correspondent? is required']")
-    private WebElementFacade whichIsThePrimaryCorrespondentErrorMessage;
+    public WebElementFacade whichIsThePrimaryCorrespondentErrorMessage;
 
     @FindBy(xpath = "//a[text()='The correspondent type must be provided']")
-    private WebElementFacade correspondentTypeMustBeProvidedErrorMessage;
+    public WebElementFacade correspondentTypeMustBeProvidedErrorMessage;
 
     @FindBy(xpath = "//a[text()='Member is required']")
-    private WebElementFacade memberIsRequiredErrorMessage;
+    public WebElementFacade memberIsRequiredErrorMessage;
 
     @FindBy(xpath = "//a[text()='The correspondent must have a type']")
-    private WebElementFacade correspondentMustHaveATypeErrorMessage;
+    public WebElementFacade correspondentMustHaveATypeErrorMessage;
 
     @FindBy(xpath = "//a[contains(@href, '#fullname-error')]")
-    private WebElementFacade correspondentNameMustBeEnteredErrorMessage;
+    public WebElementFacade correspondentNameMustBeEnteredErrorMessage;
 
     @FindBy(xpath = "//span[text()='Should the response be copied to Number 10? is required']")
-    private WebElementFacade shouldTheResponseBeCopiedN10ErrorMessage;
-
-    // Basic Methods
-
-    public void clickAddCorrespondentLink() {
-        addCorrespondentLink.click();
-    }
+    public WebElementFacade shouldTheResponseBeCopiedN10ErrorMessage;
 
-    public void clickCorrespondentIsAMember() {
-        correspondentMemberYesRadioButton.click();
```

Comment:
We should keep these as clickOns instead of having `page.clickOn(foo)` in the StepDefs probably.